src/codeu/chat/util/TokenizerTest.java

```diff
@@ -0,0 +1,27 @@
+package codeu.chat.util;
+ 
+import java.io.IOException;
+import static org.junit.Assert.*;
+import org.junit.Test;
+ 
+public final class TokenizerTest {
+  @Test
+  public void testWithQuotes() throws IOException {
+    final Tokenizer tokenizer = new Tokenizer(�\�hello world\� \�how are you\��);
+    assertEquals(tokenizer.next(), �hello world�);
+   assertEquals(tokenizer.next(), �how are you�);
+    assertEquals(tokenizer.next(), null);
+  } 
+
+  @Test
+  public void testWithNoQuotes() throws IOException {
+    final Tokenizer tokenizer = new Tokenizer(�hello world how are you�);
+    assertEquals(tokenizer.next(), �hello�);
+    assertEquals(tokenizer.next(), �world�);
+    assertEquals(tokenizer.next(), �how�);
+    assertEquals(tokenizer.next(), �are�);
+    assertEquals(tokenizer.next(), �you�);
+    assertEquals(tokenizer.next(), null);
+  }
+
+}
```

Comment:
Add one more newline at the end, just to keep this file consistent with the others